src/components/ContentSidebar/SidebarAccessStats.js

```diff
@@ -29,20 +30,24 @@ const SidebarAccessStats = ({
         edit_count: 0,
         has_count_overflowed: false
     },
-    file
+    file,
+    error,
+    intl
 }: Props) => {
     const { preview_count, comment_count, download_count, edit_count } = accessStats;
 
-    if (!preview_count && !comment_count && !download_count && !edit_count) {
+    if (!preview_count && !comment_count && !download_count && !edit_count && !error) {
         return null;
     }
 
+    const errorMessage = error ? intl.formatMessage(error) : undefined;
```

Comment:
Fine for now, but ideally you should go change AccessStates.js to take in a Node in BRUI.

Reply:
Agreed, technically it looks like it can, but the flow type says its a string